doc/manipulating_images/manipulating_images.rst

```diff
@@ -375,32 +375,31 @@ of the transformation matrix (i.e., affine).
 Image operations: creating a ROI mask manually
 ===============================================
 
-This section shows manual steps to create and further modify a ROI
-(region of interest) spatial mask. They represent a means for "data folding",
-that is, extracting and later analyzing data from a subset of voxels rather
-than the entire brain images. As a convenient side effect, this can help
-alleviate the curse of dimensionality (i.e., statistical problems that
-arise in the context of high-dimensional input variables).
+Computing Regions of Interest (ROI) mask by ourselves requires chain of image
+operations steps to follow for nicer representation of mask over specific
+target of interest. These chain of operations are easy to follow using nilearn
+and scipy Python libraries with list of functions available and these
+functions are very easy to use and we give clear guidelines how to follow each
+step sequentially starting with pre-image operations to post-image operations
+to visualization & interpretation at each step.
```

Comment:
These chains of operations are easy to set up using Nilearn and scipy Python libraries. Here we give clear guidelines about these steps, starting with pre-image operations to post-image operations. The main point is that  visualization & results checking be possible at each step.